tests/distributions/test_continuous.py

```diff
@@ -2369,6 +2369,25 @@ def seeded_weibul_rng_fn(self):
         "check_rv_size",
     ]
 
+    # See issue #7220
+    def test_rng_different_shapes(self):
+        rng = np.random.default_rng(123)
```

Comment:
Could this test simply batch one of the arguments and check len(np.unique(draws)) == draws.size?

Reply:
Something like

```python
alpha = rng.normal(size=5)  # or whatever domain is valid
beta = rng.normal(size=(3, 1))
draws = pm.draw(pm.Weibull.dist(alpha, beta), random_seed=rng)
assert len(np.unique(draws) == draws.size
```